openssh/_mapdata/_mapdata.jinja

```diff
@@ -0,0 +1,5 @@
+# yamllint disable rule:indentation rule:line-length
+# {{ grains.get('osfinger', grains.os) }}
+---
+{#- use salt.slsutil.serialize to avoid encoding errors on some platforms #}
+{{ salt['slsutil.serialize']('yaml', map, default_flow_style=False, allow_unicode=True) | regex_replace("^\s+'$", "'", multiline=True) | trim }}
```

Comment:
How about a bit of Black-inspired formatting?  Tested this locally.

```suggestion
{{ salt['slsutil.serialize'](
     'yaml',
     map,
     default_flow_style=False,
     allow_unicode=True,
   ) | regex_replace("^\s+'$", "'", multiline=True)
     | trim }}
```

Reply:
Great, i'll do that.

Reply:
As mentioned in Slack, probably better to do the `| regex_replace("^\s+'$", "'", multiline=True)` on a separate line:

https://github.com/saltstack-formulas/openssh-formula/blob/9d8228d9dbd4d6affaf5084db26ba3f852724740/openssh/libtofs.jinja#L106-L108

Reply:
Do you prefer the closing `}}` on a separate line too?

```jinja
{{ salt['slsutil.serialize'](
     'yaml',
     map,
     default_flow_style=False,
     allow_unicode=True,
   )
   | regex_replace("^\s+'$", "'", multiline=True)
   | trim
}}
```

Reply:
@baby-gnu No, I think our "standardised" Jinja (e.g. `libtofs.jinja`) has been to allow the closing to stay at the end of the last line.

Reply:
I put it on its own line, this permit cut & past lines without markers in the way ;-)